src/Core/x/xstransvc.cpp

```diff
@@ -3,12 +3,11 @@
 #include <types.h>
 #include <string.h>
 
-extern void XST_unlock_all();
-extern int32 PKRShutdown();
-extern int8* xST_xAssetID_HIPFullPath(uint32 aid, uint32* sceneID);
-extern int XST_cnt_locked();
-extern int XST_nth_locked(int param);
-
+void XST_unlock_all();
+int32 PKRShutdown();
+int8* xST_xAssetID_HIPFullPath(uint32 aid, uint32* sceneID);
+int32 XST_cnt_locked();
+int32 XST_nth_locked(int32 param1);
```

Comment:
It looks like you forgot to delete the definitions from the C++ file, as well as include the header file you  moved them to.